pkg/collector/corechecks/snmp/internal/checkconfig/config.go

```diff
@@ -206,6 +208,11 @@ func (c *CheckConfig) SetProfile(profile string) error {
 	c.ProfileDef = &definition
 	c.Profile = profile
 
+	if log.ShouldLog(seelog.TraceLvl) {
+		profileDefJSON, _ := json.Marshal(definition)
+		log.Tracef("Profile content `%s`: %+v", profile, profileDefJSON)
```

Comment:
Should we have it as a Debug log maybe? This will only get logged once.

Reply:
Yes, it's only called on SetProfile (when profile changes). So, indeed, it's should be fine to debug log instead. 

changed here: 814a08ee19